snuba/datasets/storages/transactions.py

```diff
@@ -209,6 +210,7 @@ def transactions_migrations(
             {"start_ts", "finish_ts"},
             BEGINNING_OF_TIME,
         ),
+        TransactionColumnProcessor(),
```

Comment:
This is a storage, not a dataset. You already added this to all the relevant datasets. Storages should not have it.

More in abstract, either a QueryProcessor is applied to a storage or to the dataset, a dataset contains multiple storages (which map to a table each). Storage query processors are tied to the table while dataset query processors are tied to the abstract model of the dataset. These processors do not depend on the concrete table now but on the abstract schema, so you can keep them at dataset level.

Reply:
OK I'm confused now, because earlier you said to move this into the storage.
 
> All the transformations made in this processor are storage specific (they are specific to the table we are querying) so please add it to storages/transactions.py instead (it will be also picked up automatically by the discover dataset).

Reply:
A sorry, I misread it. For some reason I thought this was the TimeSeries one. Ignore the comment.